packages/@uppy/robodog/src/addProviders.js

```diff
@@ -0,0 +1,80 @@
+const Transloadit = require('@uppy/transloadit')
+
+const remoteProviders = {
+  dropbox: require('@uppy/dropbox'),
+  'google-drive': require('@uppy/google-drive'),
+  instagram: require('@uppy/instagram'),
+  url: require('@uppy/url')
+}
```

Comment:
Hard-coding providers for simpler usage? They are indeed lightweight now 🤔 And I guess that’s what “preset” is for.

Reply:
yea, the intention is that you don't have to add script tags or anything to add more providers, and to allow users not to learn what a 'plugin' is